### PR TITLE
test(convergence): bind inferred residual proofs (#3899)

### PR DESCRIPTION
**Pull request by Sinity**
## Summary

Close the remaining convergence-proof review gaps. Every supported synthetic parser witness now validates its identity, every parser-owned text segment, authoredness, and structured tool semantics when present. The receipt also verifies that messages remain in the provider session asserted by that exact raw artifact.

## Problem

The prior receipt could preserve IDs and one text segment while losing another, lose authoredness or provider tool outcomes outside the baseline artifact, or accept a correct message set under a foreign provider session key. A persisted all-provider receipt could also omit a subsequently cataloged provider during revalidation, and an unavailable Codex ownership view had no operational signal.

## Solution

The synthetic receipt compares raw-wire role and material-origin expectations for every admitted artifact, requires every parser-owned raw text segment through the production text/block representation, compares structured tool kind, ID, error state, and exit code, and enforces the provider-specific session identity for each one-session artifact. It persists whether a receipt used the registry-default provider census, then re-censuses current providers only for that scope. Codex ownership-view SQLite errors remain fail-closed and now emit a warning. Negative tests mutate the real dispatch result for both baseline and coverage artifacts, including re-homing messages to a different raw identity and dropping one segment while retaining its message identity.

## Verification

- `POLYLOGUE_PYTEST_WORKERS=1 direnv exec . devtools test tests/unit/core/test_synthetic_wire_support.py -k 'supported_routes_validate_selected_schema_and_parser_entry_point or authoredness_loss_is_not_accepted_for_every_supported_route or tool_identity_loss_is_not_accepted or structured_tool_outcome_loss_is_not_accepted or rejects_messages_rehomed_to_another_raw_identity'`: 24 passed.
- `POLYLOGUE_PYTEST_WORKERS=1 direnv exec . devtools test tests/unit/schemas/test_inferred_corpus_manifest.py -k 'rejects_a_bound_receipt_with_a_missing_catalog_route'`: 1 passed.
- `POLYLOGUE_PYTEST_WORKERS=1 direnv exec . devtools test tests/unit/sources/test_live_batch_support.py -k 'rejects_global_fallback_when_ownership_query_errors'`: 1 passed.
- `POLYLOGUE_PYTEST_WORKERS=1 direnv exec . devtools test tests/unit/sources/test_live_batch_support.py tests/unit/core/test_synthetic_wire_support.py tests/unit/schemas/test_inferred_corpus_manifest.py -k 'codex_append_identity_rejects_global_fallback_when_ownership_query_errors or parser_witness_segment_loss_is_not_accepted_with_preserved_message_identity or default_scope_campaign_rejects_a_new_provider_during_receipt_revalidation or explicit_scope_campaign_does_not_re_census_unselected_provider or parser_witness_content_loss_is_not_accepted_with_preserved_ids_for_every_supported_route or campaign_read_rejects_wire_route_drift'`: 10 passed at `09fd3e689`.
- Pre-push `devtools verify --quick`: 24 checks passed at final head `78b45f8c9`.

## Bead disposition

| Bead | Disposition | Evidence | Remaining scope |
| --- | --- | --- | --- |
| `polylogue-rrxe4` | Partial | Exact-head production fixes and focused tests listed above | `polylogue-un60n`, `polylogue-t0m73` remain open |

### CODE REVIEW

**Comment by Sinity** [comment]
@codex review

**Comment by Sinity** [comment]
@codex review

**Comment by Sinity** [comment]
@codex review

**Comment by Sinity** [comment]
@codex review

**Comment by Sinity** [comment]
@codex review

File: polylogue/schemas/synthetic/wire_formats.py [hunk 1]
```diff
@@ -1060,213 +1081,254 @@ def build_wire_support_receipt(*, registry: object | None = None, seed: int = 20
...
+        catalog = registry.load_package_catalog(provider)  # type: ignore[attr-defined]
+        selections = tuple(
+            (package, element)
+            for package in (catalog.packages if catalog is not None else ())
+            for element in package.elements
+        )
+        if not selections:
+            package = registry.get_package(provider, version="default")  # type: ignore[attr-defined]
+            selections = ((package, None),)
+
```
[the remark this answers is not in the document]
Comment (Sinity) [reply polylogue/schemas/synthetic/wire_formats.py:1130]: Valid. Fixed in c423b684fc722a06cb6a962e8db53c6df4101988 by deduplicating the selected provider sequence with dict.fromkeys before the existing sorted iteration. The new receipt test proves duplicate provider input produces unique sorted provider and element keys.

File: tests/unit/core/test_synthetic_wire_support.py [hunk 2]
```diff
@@ -498,6 +569,50 @@ def test_claude_code_route_only_waives_unrepresentable_nested_content() -> None:
...
+    assert not final.complete
+
+    receipt = wire_formats.build_wire_support_receipt(registry=registry)
+    receipt_entry = next(
+        entry for entry in receipt.entries if (entry.provider, entry.package_version) == ("chatgpt", "v1")
+    )
+    assert receipt_entry.construct_coverage is not None
+    assert parser_relevant in receipt_entry.construct_coverage.missing_keywords
+    assert not receipt_entry.healthy
+    assert not receipt.complete
```
[the remark this answers is not in the document]
Comment (Sinity) [reply tests/unit/core/test_synthetic_wire_support.py:649]: Valid. Fixed in c423b684fc722a06cb6a962e8db53c6df4101988 by matching the support receipt on provider, package version, and selection.element_kind. The existing parser-relevant coverage assertions now inspect the exact probed element.

File: tests/infra/inferred_corpus.py [hunk 3]
```diff
@@ -544,13 +572,172 @@ def _catalog_entries(
...
+        if support_entry.status == "unsupported":
+            reason: UnsupportedCorpusReason = (
+                "unsupported_element"
+                if support_entry.reason == "catalog element is marked unsupported"
+                else "unsupported_wire_route"
+            )
+            return UnsupportedCorpusRecord(
+                reason,
+                (support_entry.reason or "route is explicitly unsupported",),
+            )
```
[the remark this answers is not in the document]
Comment (Sinity) [reply tests/infra/inferred_corpus.py:741]: Valid. Fixed in c423b684fc722a06cb6a962e8db53c6df4101988 by exporting CATALOG_ELEMENT_UNSUPPORTED_REASON from wire_formats and using that producer-consumer token in inferred-corpus refusal mapping. Unsupported-route fallback remains fail-closed.

File: tests/property/test_inferred_corpus_loop.py [hunk 4]
```diff
@@ -234,3 +405,124 @@ def test_every_supported_inferred_element_reaches_convergence_and_red_twin(
...
+            archive_root / "ops.db",
+            stage="insights",
+            subject_type="session_id",
+            subject_id=session_id,
+            retry_at="1970-01-01T00:00:00+00:00",
+        )
+
+    assert _run_retry_in_fresh_process(archive_root / "index.db") == len(session_ids)
+    assert CursorStore(archive_root / "index.db").list_convergence_debt(limit=100) == []
+    assert archive_snapshot(archive_root, session_ids=session_ids) == baseline
```
[the remark this answers is not in the document]
Comment (Sinity) [reply tests/property/test_inferred_corpus_loop.py:467]: Valid. Fixed in c423b684fc722a06cb6a962e8db53c6df4101988 by marking the fresh-process debt recovery test uses_real_clock. Its debt timestamp and child-process retry drain intentionally use wall-clock time.

File: tests/infra/inferred_corpus.py [hunk 5]
```diff
@@ -756,13 +968,44 @@ def _validate_inference_handoff(
...
             schema=live_schema if isinstance(live_schema, dict) else None,
             wire_format=PROVIDER_WIRE_FORMATS.get(provider),
             construct_support=live_constructs,
+            support_entry=_wire_support_entry_from_manifest(
+                manifest,
+                provider=provider,
+                package_version=package.version,
+                element_kind=element.element_kind,
+            ),
+            support_receipt_bound=manifest.wire_support_receipt is not None,
```
[the remark this answers is not in the document]
Comment (Sinity) [reply tests/infra/inferred_corpus.py:1010]: Valid. Fixed in c423b684fc722a06cb6a962e8db53c6df4101988. The campaign validator builds one keyed persisted wire-entry index before the catalog loop and uses exact provider, package version, and element lookups instead of rescanning the receipt.

File: tests/infra/inferred_corpus.py [hunk 6]
```diff
@@ -756,13 +968,44 @@ def _validate_inference_handoff(
...
+            expected_unsupported.add(
+                (
+                    provider,
+                    package.version,
+                    element.element_kind,
+                    "nonrepresentable" if schema_reason == "unsupported_json_schema_construct" else "unsupported",
+                    schema_reason,
+                    schema_details,
+                )
+            )
```
[the remark this answers is not in the document]
Comment (Sinity) [reply tests/infra/inferred_corpus.py]: Valid. Fixed in c423b684fc722a06cb6a962e8db53c6df4101988. _schema_unsupported_reason now owns the shared precedence and is called by both compiler and campaign validation. The unloaded-schema plus missing-wire-format regression pins missing_schema.

File: polylogue/sources/live/batch.py [hunk 7]
```diff
@@ -3486,10 +3544,19 @@ def _append_payload_for_provider(
...
+                provider=provider.value,
+                package_version="live",
+                element_kind="session_record_stream",
+                stable_session_identity=identity is not None,
+            )
+            if capability.status != "supported":
                 return None
+        else:
+            identity = None
+        if provider is Provider.CODEX:
```
[the remark this answers is not in the document]
Comment (Sinity) [reply polylogue/sources/live/batch.py:3563]: Valid. Fixed in c423b684fc722a06cb6a962e8db53c6df4101988. The append planner now passes the canonical current provider origin into the archived identity lookup, and the real source/index-tier regression test refuses a wrong-origin row at the same path.

**Comment by Sinity** [comment]
@codex review

File: tests/infra/inferred_corpus.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (Sinity) [reply tests/infra/inferred_corpus.py:1010]: Follow-up in signed commit `3d17c7100a7f8f877d0a3f555d08f33cdbc73c95`: `test_campaign_indexes_persisted_wire_support_entries_once` wraps the production `_wire_support_entry_index` and `_wire_support_entry_from_payload` seams, requiring one index invocation and exactly one parse per persisted receipt entry. The focused proof passes; reverting to the old per-element scan fails the index-invocation assertion.

**Comment by Sinity** [comment]
@codex review

**Comment by Sinity** [comment]
@codex review

Please review exact PR head `5caf39d7ea5e38ca0db78dfe55ac8c8281dbe367` only. Ignore earlier review snapshots tied to prior heads.

**Comment by Sinity** [comment]
@codex review

File: tests/infra/inferred_corpus.py [hunk 8]
```diff
@@ -544,13 +571,172 @@ def _catalog_entries(
...
+        raise ValueError("wire_support_receipt entries must be a list")
+    for raw_entry in raw_entries:
+        if not isinstance(raw_entry, Mapping):
+            raise ValueError("wire_support_receipt entries must be objects")
+        if (
+            raw_entry.get("provider"),
+            raw_entry.get("package_version"),
+            raw_entry.get("element_kind"),
+        ) == (provider, package_version, element_kind):
+            return _wire_support_entry_from_payload(raw_entry)
```
[the remark this answers is not in the document]
Comment (Sinity) [reply tests/infra/inferred_corpus.py]: Continuation at pushed head e78afd8f27599a673dbf9505aeb971e027bd6dc8 tightens the parser witness boundary: _parser_artifact_evidence now requires parsed message or block content to occur in the provider-specific raw evidence node as well as preserving identity binding. The focused ID-preserving invented-content red twin fails if that content binding is removed.

File: polylogue/schemas/synthetic/wire_formats.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sinity) [reply polylogue/schemas/synthetic/wire_formats.py:1130]: Continuation also closes the identity and ordering residuals. WireSupportReceipt construction, manifest compilation, persisted loading, and indexing now reject duplicate (provider, package_version, element_kind) keys, while receipt construction sorts provider, package version, and element kind. The conflicting-duplicate and reordered-catalog proofs pass in signed commits 454bc35c604f7dcab3504f98381afcad7d66a61a and e78afd8f27599a673dbf9505aeb971e027bd6dc8.

File: polylogue/sources/live/batch.py (hunk 7 above)
[the remark this answers is not in the document]
Comment (Sinity) [reply polylogue/sources/live/batch.py:3563]: Continuation at exact head e78afd8f27599a673dbf9505aeb971e027bd6dc8 extends the origin guard to both sessions.origin and durable raw_sessions.origin, including the Codex fallback join. Both mixed directions, index-Codex/source-Claude and index-Claude/source-Codex, refuse append authorization.

File: tests/infra/inferred_corpus.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (Sinity) [reply tests/infra/inferred_corpus.py:1010]: The continuation keeps the support-entry index one-time and adds duplicate-key rejection before compilation and at persisted load/index boundaries. The existing one-index call-count proof still passes, and the conflicting duplicate fixture proves first/last silent collapse cannot return.

File: tests/property/test_inferred_corpus_loop.py [hunk 9]
```diff
@@ -234,3 +405,125 @@ def test_every_supported_inferred_element_reaches_convergence_and_red_twin(
...
+    before = archive_snapshot(archive_root, session_ids=session_ids)
+
+    receipt = rebuild_retained_raw_index(archive_root)
+    _converge_existing_archive(archive_root)
+
+    assert receipt.raw_session_count == receipt.selected_raw_count > 0
+    assert archive_snapshot(archive_root, session_ids=session_ids) == before
+
+
+@pytest.mark.uses_real_clock
```
[the remark this answers is not in the document]
Comment (Sinity) [reply tests/property/test_inferred_corpus_loop.py]: Fixed in signed commit e78afd8f27599a673dbf9505aeb971e027bd6dc8. The marker now explains that fresh-process debt recovery crosses a subprocess wall-clock retry boundary. The focused test and final quick gate pass.

**Comment by Sinity** [comment]
@codex review

File: polylogue/schemas/synthetic/wire_formats.py [hunk 10]
```diff
@@ -384,6 +389,12 @@ def _route_nonrepresentable_reasons(provider: str, missing_keywords: Collection[
     }
     for keyword in missing_keywords:
         path = keyword.split("@", 1)[1] if "@" in keyword else "$"
+        if provider == "chatgpt" and package_version == "v1":
+            reasons[keyword] = (
+                "ChatGPT v1 parser route retains normalized conversation fields but does not represent "
+                "export-only media metadata at this exact package selection"
+            )
+            continue
```
[the remark this answers is not in the document]
Comment (Sinity) [reply polylogue/schemas/synthetic/wire_formats.py]: Reviewed at exact head e0cff49ea8d07f4df58f5172459e54672fb301f5. The complete parser witness gate now compares the full parser-relevant identity and content multiset, so a parser omission cannot obtain healthy evidence. The partial-output red-twin test exercises this boundary. This older thread is addressed by the current implementation.

File: polylogue/sources/live/batch.py [hunk 11]
```diff
@@ -342,6 +342,64 @@ def _write_codex_thread_state_evidence(
...
+    """Resolve append support from the live route's identity contract."""
+    if provider not in {"codex", "claude-code"}:
+        return AppendCapabilityReceipt(
+            provider=provider,
+            package_version=package_version,
+            element_kind=element_kind,
+            status="unsupported",
+            reason="live append route supports only Codex and Claude Code JSONL identity contracts",
+        )
+    if provider == "codex" and not stable_session_identity:
```
[the remark this answers is not in the document]
Comment (Sinity) [reply polylogue/sources/live/batch.py]: Reviewed at exact head e0cff49ea8d07f4df58f5172459e54672fb301f5. Claude stable identity remains required by the capability route, and the append path rejects an absent archived identity. The existing Claude capability and tail-matching tests cover that contract. This older thread is addressed by the current branch.

File: tests/infra/inferred_corpus.py [hunk 12]
```diff
@@ -763,6 +814,8 @@ def _validate_inference_handoff(
             schema=live_schema if isinstance(live_schema, dict) else None,
             wire_format=PROVIDER_WIRE_FORMATS.get(provider),
             construct_support=live_constructs,
+            support_entry=None,
+            support_receipt_bound=False,
```
[the remark this answers is not in the document]
Comment (Sinity) [reply tests/infra/inferred_corpus.py]: Reviewed at exact head e0cff49ea8d07f4df58f5172459e54672fb301f5. Campaign validation uses the bound wire-support decision and preserves explicit unsupported-route outcomes. The existing campaign round-trip and route-drift tests cover this persisted authority boundary. This older thread is addressed by the current branch.

File: polylogue/storage/insights/session/latency_profiles.py [hunk 13]
```diff
@@ -60,12 +60,13 @@ def build_session_latency_profile_record(
...
     )
+    source_sort_timestamp = profile.updated_at or profile.created_at
     return SessionLatencyProfileRecord(
         session_id=SessionId(str(session.id)),
         materializer_version=SESSION_INSIGHT_MATERIALIZER_VERSION,
         materialized_at=built_at,
-        source_updated_at=_iso_datetime(session.updated_at),
-        source_sort_key=float(session.updated_at.timestamp()) if session.updated_at is not None else None,
+        source_updated_at=_iso_datetime(profile.updated_at),
+        source_sort_key=float(source_sort_timestamp.timestamp()) if source_sort_timestamp is not None else None,
```
[the remark this answers is not in the document]
Comment (Sinity) [reply polylogue/storage/insights/session/latency_profiles.py:70]: Reviewed at exact head e0cff49ea8d07f4df58f5172459e54672fb301f5. The latency record retains the fallback sort key used for created-only sessions, and materialization uses that latency provenance. The fallback-sort-key and created-without-updated tests cover the readiness boundary. This older thread is addressed by the current branch.

File: polylogue/storage/insights/session/profiles.py [hunk 14]
```diff
@@ -349,16 +349,19 @@ def build_session_profile_record(
     enrichment = session_enrichment_payload(profile, analysis)
     evidence_search_text = profile_evidence_search_text(profile)
     inference_search_text = profile_inference_search_text(profile)
-    source_updated_at = profile.updated_at.isoformat() if profile.updated_at else None
+    source_sort_timestamp = profile.updated_at or profile.created_at
+    source_updated_at = source_sort_timestamp.isoformat() if source_sort_timestamp else None
```
[the remark this answers is not in the document]
Comment (Sinity) [reply polylogue/storage/insights/session/profiles.py]: Reviewed at exact head e0cff49ea8d07f4df58f5172459e54672fb301f5. Creation-only sessions retain a source sort key for readiness but do not invent provider update or high-water-mark provenance. The created-without-updated materialization test covers that distinction. This older thread is addressed by the current branch.

File: tests/infra/inferred_corpus.py (hunk 8 above)
[the remark this answers is not in the document]
Comment (Sinity) [reply tests/infra/inferred_corpus.py]: Reviewed at exact head e0cff49ea8d07f4df58f5172459e54672fb301f5. The current parser witness path retains provider-specific content and identity binding, and now also requires complete coverage of all expected parsed messages. The parser witness tests cover both boundaries. This older thread is superseded by the exact-head coverage repair.

File: polylogue/sources/live/batch.py [hunk 15]
```diff
@@ -3486,10 +3544,19 @@ def _append_payload_for_provider(
         NEW writes going forward, per polylogue-u19l's scope.
         """
         provider = Provider.from_string(canonical_acquisition_provider(source_name, source_name=source_name))
-        if provider is Provider.CODEX:
+        if provider in {Provider.CODEX, Provider.CLAUDE_CODE}:
             identity = self._existing_provider_session_id(path)
```
[the remark this answers is not in the document]
Comment (Sinity) [reply polylogue/sources/live/batch.py]: Reviewed at exact head e0cff49ea8d07f4df58f5172459e54672fb301f5. The existing Claude append route carries the resolved identity through its tail validation path, avoiding the duplicate lookup. The live read-amplification coverage remains in the branch. This older thread is addressed by the current branch.

File: polylogue/schemas/synthetic/wire_formats.py [hunk 16]
```diff
@@ -1060,213 +1081,252 @@ def build_wire_support_receipt(*, registry: object | None = None, seed: int = 20
...
-                    provider=provider,
-                    status=route.status,
-                    reason=route.reason,
-                    package_version=package_version,
-                    element_kind=element_kind,
-                    schema_valid=None,
-                    parsed_session_count=0,
-                    parsed_message_count=0,
-                    construct_coverage=None,
+            if element is not None and not element.supported:
```
[the remark this answers is not in the document]
Comment (Sinity) [reply polylogue/schemas/synthetic/wire_formats.py:1673]: Reviewed at exact head e0cff49ea8d07f4df58f5172459e54672fb301f5. Missing provider routes are classified at provider scope before unsupported catalog elements are skipped. The existing missing-route receipt test covers this census boundary. This older thread is addressed by the current branch.

File: polylogue/schemas/synthetic/wire_formats.py [hunk 17]
```diff
@@ -1060,213 +1081,252 @@ def build_wire_support_receipt(*, registry: object | None = None, seed: int = 20
     from polylogue.schemas.validator import SchemaValidator, ValidationResult
     from polylogue.sources.dispatch import parse_payload, require_positive_conversational_evidence
 
-    catalog_providers = tuple(sorted(registry.list_providers()))  # type: ignore[attr-defined]
+    catalog_providers = tuple(sorted(providers or registry.list_providers()))  # type: ignore[attr-defined]
```
[the remark this answers is not in the document]
Comment (Sinity) [reply polylogue/schemas/synthetic/wire_formats.py]: Reviewed at exact head e0cff49ea8d07f4df58f5172459e54672fb301f5. An explicit empty provider selection remains distinct from a None default and does not expand to the full registry. The existing provider-selection test covers this scope boundary. This older thread is addressed by the current branch.

File: tests/property/test_inferred_corpus_loop.py (hunk 9 above)
[the remark this answers is not in the document]
Comment (Sinity) [reply tests/property/test_inferred_corpus_loop.py]: Reviewed at exact head e0cff49ea8d07f4df58f5172459e54672fb301f5. The real-clock marker names the fresh-process subprocess and debt-retry wall-clock boundary. The focused test and quick gate pass with that documented exemption. This older thread is addressed by the current branch.

File: polylogue/schemas/synthetic/wire_formats.py [hunk 18]
```diff
@@ -1060,218 +1100,269 @@ def build_wire_support_receipt(*, registry: object | None = None, seed: int = 20
...
+                            artifact_evidence=artifact_evidence,
+                        )
+                    )
+                    if (
+                        artifact_sessions
+                        and artifact_evidence
+                        and artifact_validation_error is None
+                        and parse_error is None
+                    ):
+                        payloads.extend(payload_items)
```
[the remark this answers is not in the document]
Comment (Sinity) [reply polylogue/schemas/synthetic/wire_formats.py:1851]: Reviewed at exact head e0cff49ea8d07f4df58f5172459e54672fb301f5. The parser witness gate now requires the complete expected message identity and content multiset, and the partial-output red-twin test fails when that gate is removed. This current-head finding is addressed.

File: polylogue/storage/insights/session/rebuild.py [hunk 19]
```diff
@@ -1461,16 +1469,27 @@ def _stamp_bundle_materialization(conn: sqlite3.Connection, bundle: SessionInsig
...
+        stamp_source_sort_key_ms = source_sort_key_ms
+        stamp_input_high_water_mark_ms = input_high_water_mark_ms
+        stamp_input_high_water_mark_source = profile.input_high_water_mark_source
+        if insight_type == "latency":
+            stamp_source_updated_at_ms = _epoch_ms_or_none(latency.source_updated_at)
+            stamp_source_sort_key_ms = (
+                int(latency.source_sort_key * 1000) if latency.source_sort_key is not None else None
+            )
+            stamp_input_high_water_mark_ms = _epoch_ms_or_none(latency.input_high_water_mark)
+            stamp_input_high_water_mark_source = latency.input_high_water_mark_source
```
[the remark this answers is not in the document]
Comment (Sinity) [reply polylogue/storage/insights/session/rebuild.py:1482]: Reviewed at exact head e0cff49ea8d07f4df58f5172459e54672fb301f5. The ordinary latency rebuild now passes the profile provider high-water mark and source into the latency record, and the real convergence test asserts both the row and materialization marker. This current-head finding is addressed.

**Comment by Sinity** [comment]
@codex review

Please perform a fresh exact-head review of PR #3899 at commit e0cff49ea8d07f4df58f5172459e54672fb301f5 only. Recheck the three residual findings in the current production diff and their anti-vacuity tests: complete parser witness coverage, Codex index-only identity authority with wrong-origin rejection, and provider high-water-mark propagation into ordinary latency materialization. Treat older inline threads as answered only if the exact current head supports that conclusion. Do not review an earlier commit.

**Comment by Sinity** [comment]
@codex review

Please review the exact pushed head `1e559a992fb92e838801e085ed26f72fae8d3336`, including the mixed-origin Codex identity regression test and the refreshed scope carrier.

**Comment by Sinity** [comment]
@codex review

Please review exact pushed head `cd790ab28a2e73e91e801537508cf7408664f38b` only against the refreshed scope carrier. Review the new wire-coverage diagnostics and the prior owner/coverage authority findings.
